clang/lib/AST/ASTContext.cpp

```diff
@@ -2200,6 +2206,9 @@ TypeInfo ASTContext::getTypeInfoImpl(const Type *T) const {
     Align = 8;                                                                 \
     break;
 #include "clang/Basic/WebAssemblyReferenceTypes.def"
+    case BuiltinType::AMDGPUBufferRsrc:
+      Width = 128;
+      Align = 128;
```

Comment:
If we were exposing the pointer, it would be 160/192

Reply:
Do we want to expose it as a pointer, a sizeless object, or a vector of 160 bits?

Reply:
Sizeless object is probably the safest option for now. It shouldn't be a vector. I foresee more difficulty introducing a wide pointer, and if we change our minds on how fat pointers should work it's harder to change the size later